pkg/lib/verifier/rule_builder_util.go

```diff
@@ -17,112 +18,174 @@ import (
 	"github.com/pkg/errors"
 )
 
-func getPcrMatchesConstantRules(pcrs []types.PcrIndex, flavor *hvs.Flavor, marker common.FlavorPart) ([]rules.Rule, error) {
+//getPcrMatchesConstantRules method will create PcrMatchesConstantRule and return the rule
+//return nil if error occurs
+func getPcrMatchesConstantRules(pcrs []types.PcrIndex, flavor *hvs.Flavor, pcrLogData *types.PCRS, marker common.FlavorPart) ([]rules.Rule, error) {
 
 	var results []rules.Rule
```

Comment:
Using variable names as 'rules' instead of 'results' is showing error because we are using package name called 'rules'. Hence it is conflicting. Can we use some other name like 'pcrRules'?